doc/README.md

```diff
@@ -73,6 +73,7 @@ The Bitcoin repo's [root README](/README.md) contains relevant information on th
 - [Assets Attribution](assets-attribution.md)
 - [Assumeutxo design](assumeutxo.md)
 - [bitcoin.conf Configuration File](bitcoin-conf.md)
+- [CJDNS Support](cjdns.md)
```

Comment:
This link is dead.

Reply:
Good catch. The commit adding that file is missing.

Reply:
Thanks! Fixed the link at https://github.com/jonatack/bitcoin/blob/23.x-onlynet-help-and-doc-improvements/doc/README.md#miscellaneous by adding missing commits from #24555 and checked the other backports.